Adds web crawler example to Knowledge Base doc (#6176)
elastic/security-docs: docs/AI-for-security/knowledge-base.asciidoc

+ 
NOTE: Remember, each index added to Knowledge Base must have at least one semantic text field.
+
.. Under **Field type**, select `Semantic text`. Under **Select an inference endpoint**, select `elastic-security-ai-assistant-elser2`. Click **Add field**, then **Save mapping**.
. Go to the **Scheduling** tab. Enable the **Enable recurring crawls with the following schedule** setting, and define your desired schedule.
. Go to the **Manage Domains** tab. Select the domain associated with your new web crawler, then go the its **Crawl rules** tab and click **Add crawl rule**. For more information, refer to {enterprise-search-ref}/crawler-extraction-rules.html[Web crawler content extraction rules].
.. Under **Policy**, select `Allow`. Under **Rule**, select `Contains`. Under **Path pattern**, enter your path pattern, for example `threat-intelligence`. Click **Save**.
.. Click **Add crawl rule** again. Under **Policy**, select `Disallow`. Under **Rule**, select `Regex`. Under **Path pattern**, enter `.*`. Click **Save**. Make sure this rule appears below the rule created in the previous step on the list.

> .. Click **Crawl**, then **Crawl all domains on this index**. A message appears that says "Successfully scheduled a sync, waiting for a connector to pick it up". 

Reviewer: This is slightly easier to maintain because you don't have to update the text if/when the copy changes.

```suggestion
.. Click **Crawl**, then **Crawl all domains on this index**. A success message appears. 
```

> . The crawl process will take longer for larger data sources. Once it finishes, your new web crawler's index will contain documents provided by the crawler.

Reviewer: This seems to be the result of an action, not so much a new step. Maybe add this to the end of step 5c, or at the end of step 5?
